File: 1319-unique-number-of-occurrences/unique-number-of-occurrences.py
from collections import Counter
class Solution:
    def uniqueOccurrences(self, arr: List[int]) -> bool:
        frequency = Counter(arr)
        done = []
        for count in frequency.values():
            if count not in done:
                done.append(count)
            else:
                return False
        return True

        

File: 1319-unique-number-of-occurrences/unique-number-of-occurrences.go
func uniqueOccurrences(arr []int) bool {
    frequency := map[int]int{}
    for _, num := range arr {
        _, found := frequency[num]
        if found {
            frequency[num]+=1
        } else {
            frequency[num]=1
        }
    }
    done := []int{}
    for _, count := range frequency {
        if (slices.Contains(done, count)) {
            return false
        } else {
            done = append(done, count)
        }
    }
    return true
}